overlay/sbin/k3os-upgrade

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+set -x
+
+PROC=$(uname -m)
+
+# Take a parameter of the version number (i.e. v0.4.0) if it is given, otherwise use latest
+if [ -z $K3OS_VERSION ]
+then
+	K3OS_VERSION="latest"
+fi
+
+if [ $PROC == "x86_64" ]
+then
+	ARCH="amd64"
+elif [ $PROC == "aarch64" ]
+then
+	ARCH="arm64"
+elif [ $PROC == "armv7l" ]
+then
+	ARCH="arm"
+else
+	echo "Unsupported CPU architecture."
+	exit 0
+fi
+
+echo "Upgrading k3os to ${VERSION}"
+
+cd /k3os/system/
+mount -o remount rw .
```

Comment:
Given the remount (and the permissioned untar/copy below) please implement a check at the top of this script to make sure we are running as `root` and then nix any usage of `sudo`.

Reply:
Done.